.env.example

```
@@ -1,7 +1,7 @@
 ASPNETCORE_ENVIRONMENT=Development
 DOTNET_ENVIRONMENT=Development
 
-ConnectionStrings__Default="Host=localhost;Port=5432;Database=degen;Username=postgres;Password=postgres"
+ConnectionStrings__Default="Host=localhost;Port=5432;Database=postgres;Username=postgres;Password=postgres"
```

dotenv-linter (4.0.0)

[warning] [LowercaseKey] The ConnectionStrings__Default key should be in uppercase

(LowercaseKey)